classes/plugininfo/factor.php

```diff
@@ -82,6 +82,48 @@ public static function get_enabled_factors() {
         return $return;
     }
 
+    /**
+     * Finds enabled factors for user by userid.
+     *
+     * @param int $userid user id.
+     * @return array of enabled factors for given user.
+     */
+    public static function get_enabled_user_factors($userid) {
```

Comment:
Is this supposed to return all factors for a user, or all factors for this particular type of factor for a particular user?

Reply:
I agree that it's not a best choice for method name. Amended to make it more clear